src/protocol/RemotingCommand.cpp

```diff
@@ -250,4 +290,15 @@ void RemotingCommand::addExtField(const string& key, const string& value) {
   m_extFields[key] = value;
 }
 
+std::string RemotingCommand::ToString() {
```

Comment:
explicitly specify this method as const ?

Reply:
ok